src/jacobi.jl

```diff
@@ -76,8 +78,8 @@ singularitiesbroadcast(F::Function, G::Function, V::SubQuasiArray, K) = singular
 singularitiesbroadcast(F, V::Union{NoSingularities,SubQuasiArray}...) = singularitiesbroadcast(F, map(_parent,V)...)[_parentindices(V...)...]
 
 
-singularitiesbroadcast(::typeof(*), ::LegendreWeight, b::JacobiWeight) = b
-singularitiesbroadcast(::typeof(*), a::JacobiWeight, ::LegendreWeight) = a
+#singularitiesbroadcast(::typeof(*), ::LegendreWeight, b::JacobiWeight) = b
+#singularitiesbroadcast(::typeof(*), a::JacobiWeight, ::LegendreWeight) = a
```

Comment:
This too, it should not be commented out

Reply:
Just delete the comment out and push